monkey/monkey_island/cc/ui/src/components/Main.tsx

```diff
@@ -82,74 +78,12 @@ class AppComponent extends AuthComponent {
           });
         })
     }
-
-    if (res) {
-      this.setMode()
-        .then(() => {
-          if (this.state.islandMode === "unset") {
-            return
-          }
-
-          // update status: report generation
-          this.authFetch('/api/report-generation-status', {}, false)
```

Comment:
Do we really need to remove these steps? Aren't these needed ?

Reply:
Fixed